src/map/packets/unity_start.cpp

```diff
@@ -0,0 +1,61 @@
+﻿#include "common/socket.h"
+
+#include "unity_start.h"
+
+#include "entities/charentity.h"
+
+CUnityStartPacket::CUnityStartPacket()
+{
+    this->setType(0x75);
+    this->setSize(0xAC);
+
+    // ref<uint8>(0x25) = ; // Flags value with multiple usages.
```

Comment:
Any chance you can expand on these addresses you've commented out with any notes and findings you might have, even if they aren't implemented this time around?

Reply:
I actually haven't figured out or done much testing on what these 2 bytes do. The comments about them being flag values are courtesy of Atom0s.

Reply:
The only thing I really noticed about them is that 0x25 changes frequently in Dynamis.

Reply:
If it's usable for unity/DI/current stuff as-is, then don't worry about trying to find more notes